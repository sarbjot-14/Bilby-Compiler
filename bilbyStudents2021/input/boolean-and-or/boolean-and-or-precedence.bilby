main{
	% comparison should be done first, then and, then or
	print (true && true) == (2>1) $n;
	%print true && true == 2>1 $n;
	
}