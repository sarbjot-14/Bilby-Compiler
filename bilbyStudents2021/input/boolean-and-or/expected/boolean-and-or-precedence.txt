Couldn't read file "/home/sarbjot/Desktop/cmpt379/bilbyCompiler/bilbyStudents2021/input/boolean-and-or/output/boolean-and-or-precedence.asm"
